WinPort/WinPort.h

```diff
@@ -21,7 +21,7 @@ extern "C" {
 	void WinPortHelp();
 	const wchar_t *WinPortBackend();
 
-	///console API
```

Comment:
actually triple / is not a mistake - it produces special coloring:
![Screenshot_20230312_182100](https://user-images.githubusercontent.com/20957349/224554380-a11014da-285e-412d-9ee7-7cd3c53dcf5c.png)

Reply:
fixed that